### PR TITLE
Add DVGP backing support for vcd_nsxt_network_imported

### LINKED ISSUE #865: vcd_nsxt_network_imported  should support Distributed virtual port group backed networks
Hello,

### Terraform Version
Terraform v1.2.2
on windows_amd64
+ provider registry.terraform.io/vmware/vcd v3.6.0

### Affected Resource(s)
vcd_nsxt_network_imported

### Terraform Configuration Files
Can't add these currently

### Debug Output
Can't add these currently

### Panic Output
Can't add these currently

### Expected Behavior
When I import a vcd_nsxt_network_imported from a vdc the resource is found and imported so assume I'm using the correct call.
Although, this was an imported "dvswitch from vCenter" and not an NSX-T logical switch, so is a standard port group.

### Actual Behavior
On import of the already created resource into terraform the tfstate has "nsxt_logical_switch_name": null
If I run the script with this value I get:

│ Error: Missing required argument
│
│   with module.name.vcd_nsxt_network_imported.vcd_nsxt_network_imported,
│   on modules\vcd_nsxt_network_imported\main.tf line 27, in resource "vcd_nsxt_network_imported" "vcd_nsxt_network_imported":
│   27:   nsxt_logical_switch_name = null
│
│ The argument "nsxt_logical_switch_name" is required, but no definition was found.

If I change the name it wants to recreate the resource which isn't an option

### LINKED ISSUE #951: use the id instead of the name for imported nsx-t networks
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

We are creating segments in the NSX-T Manager and importing them into VCD in the same terraform stack. After creating a segment in the NSX-T Manager, we are using `nsxt_logical_switch_name` to import the correct segment. Sometimes we need to change the name of the segments, which in turn updates the `nsxt_logical_switch_name` field. This forces a replacement of the imported network, which is kind of bad.
I would propose to improve the `vcd_nsxt_network_imported` resource to also accept `nsxt_locgical_switch_id` as an argument, so I can directly input the id, which does not change.

### New or Affected Resource(s)

* vcd_nsxt_network_imported

### Terraform Configuration (if it applies)

```hcl
resource "nsxt_policy_segment" "segment1" {
  display_name        = "segment1"
  description         = "Terraform provisioned Segment"
  connectivity_path   = nsxt_policy_tier1_gateway.mygateway.path
  transport_zone_path = data.nsxt_policy_transport_zone.overlay_tz.path

  subnet {
    cidr = "12.12.2.1/24"
  }
}

resource "vcd_nsxt_network_imported" "segment1" {
  org      = "my-org"
  owner_id = data.vcd_org_vdc.main.id

  name        = nsxt_policy_segment.segment1.display_name
  description = "My NSX-T VDC Imported network type"

  nsxt_logical_switch_id = nsxt_policy_segment.segment1.id

  gateway       = "12.12.2.1"
  prefix_length = 24
}
```

### PR DESCRIPTION
**Pull request by Didainius**
This PR adds Distributed Virtual Port Group backing support for `vcd_nsxt_network_imported`. 

Closes #865 (adds support for DVPG backed imported networks)
Closes #951 (there is unfortunately still no support for it in current versions)

Note. A new `Dvpg` test config is added.

Additionally, this PR adds a `make cleanup` target.

### CODE REVIEW

File: website/docs/r/nsxt_network_imported.html.markdown [hunk 1]
```diff
@@ -88,12 +121,13 @@ Static IP Pools  support the following attributes:
 * `end_address` - (Required) The final address in the IP Range
 
 ## Attribute Reference
-* `nsxt_logical_switch_id` - ID of an existing NSX-T segment
+* `nsxt_logical_switch_id` - ID of NSX-T logical switch used by this network
+* `dvpg_id` - ID of Distributed Virtual Port Group used by this network
```
Comment (lvirbalas) [inline website/docs/r/nsxt_network_imported.html.markdown:125]: Should we mention that it comes from vSphere here?

File: website/docs/r/nsxt_network_imported.html.markdown [hunk 2]
```diff
@@ -64,12 +94,15 @@ and inherited from provider configuration)
 * `vdc` - (Deprecated; Optional) The name of VDC to use. **Deprecated**  in favor of new field
   `owner_id` which supports VDC and VDC Group IDs.
 * `name` - (Required) A unique name for the network
-* `nsxt_logical_switch_name` - (Required) Unique name of an existing NSX-T segment. 
+* `nsxt_logical_switch_name` - (Optional) Unique name of an existing NSX-T segment.
```
Comment (lvirbalas) [inline website/docs/r/nsxt_network_imported.html.markdown:97]: What about stating that the user should provide either `nsxt_logical_switch_name` OR `dvpg_name`?

File: website/docs/r/nsxt_network_imported.html.markdown (hunk 1 above)
Comment (Didainius) [reply website/docs/r/nsxt_network_imported.html.markdown:125]: This is tricky to state as most probably the ID would not exactly match vSphere ID as we have seen already

**Review by lvirbalas — APPROVED** [review]

**Review by adambarreiro — APPROVED** [review]
LGTM!

**Pull request merged**